lib/githubchart/svg.rb

```diff
@@ -21,6 +22,14 @@ def svg
       chart.to_s
     end
 
+    def svg_square
+      grid = matrix.minor(0..(matrix.row_size-1), (matrix.column_size - 7)..(matrix.column_size - 1))
```

Comment:
So, "matrix" is itself a method, which returns `@stats` pumped into a Matrix, from githubchart.rb. So what this does calls that method like 5 times